sdk/core/core-lro/test/public/lro.spec.ts

```diff
@@ -3172,7 +3212,7 @@ matrix(
               implName,
               throwOnNon2xxResponse: true,
             });
-            await expect(poller).rejects.toThrow(errMsg);
+            await expect(poller).to.be.rejectedWith(errMsg);
```

Comment:
feels weird to use `expect` with `chaiAsPromised` syntax when the `rejects` syntax is built into vitest natively: https://vitest.dev/api/expect.html#rejects

Reply:
Yeah importing chai-as-promised breaks the `.rejects` API, so I had to choose one. I reverted the chai-as-promised change in https://github.com/Azure/azure-sdk-for-js/pull/33363/commits/a5a14816c9223e76bad3d51e240e5419d0176458